.pre-commit-config.yaml

```diff
@@ -1,77 +1,29 @@
-#
-# Run pre-commit hooks. You can run them without installing
-#  the hook with
-#
-# $ pre-commit run --all-files
-#
-# See https://pre-commit.com for more information
-# See https://pre-commit.com/hooks.html for more hooks
 repos:
--   repo: https://github.com/pre-commit/pre-commit-hooks
-    rev: v4.3.0
-    hooks:
-    -   id: trailing-whitespace
-    -   id: end-of-file-fixer
-    -   id: check-yaml
-        args: [--allow-multiple-documents]
-    -   id: check-added-large-files
-        args:
-        - "--maxkb=4000"
-- repo: https://github.com/myint/autoflake
-  rev: v1.4
-  hooks:
-    - id: autoflake
-      args:
-        - --in-place
-        - --remove-unused-variables
-        - --remove-all-unused-imports
--   repo: https://github.com/psf/black
-    rev: 22.6.0
-    hooks:
-    -   id: black
-- repo: https://github.com/pycqa/isort
-  rev: 5.10.1
-  hooks:
-    - id: isort
-      name: isort (python)
-      # Use black profile for isort to avoid conflicts
-      #   see https://github.com/PyCQA/isort/issues/1518
-      args: ["--profile", "black"]
-    - id: isort
-      name: isort (cython)
-      types: [cython]
-    - id: isort
-      name: isort (pyi)
-      types: [pyi]
-- repo: https://gitlab.com/pycqa/flake8
-  rev: 3.9.2
```

Comment:
If the repository contains python files, these checks should probably stay.
If the repository does not contain python files, they can be removed.